grizli/prep.py

```diff
@@ -1824,7 +1858,7 @@ def make_SEP_catalog(root='',
             # tab = tab[~bad]
 
             # Correction for flux outside Kron aperture
-            tot_corr = get_wfc3ir_kron_tot_corr(tab, drz_filter, 
+            tot_corr = get_kron_tot_corr(tab, drz_filter, 
```

Comment:
I agree to rename the function, but the function itself will have to be amended in some detail to work for the JWST instruments

Reply:
This was actually you, I didn't rename this function